n //n° de l'objet
O // nb total d'objet (même O que le programme)

Il faut que la fonction commence quand n=O-1 ( quand le dernier objet est affiché à l'écran)
quand n=O-1 ==> la fonction se lance

# Théorie 1

1 =>9
2=>8
3=>7
.
.
.

O=10
n=1
O-n=9

n=2
8

#Théorie 1 ## phase test
### test 1

Un test peut-être inutile
On change les arguments de la fonction precedent => suivant
On test pour O=10 // valeur de base
ecart 300
  compilation OK
  execution BUG
  
ECHEC

### test 2

On change seulement la maniere d'appeler la fonctio dans le main
on garde ce code 			obj[n] = retour_au_point_de_depart_objet(obj[n], obj[n-1]); // code pour faire appel à la fonc.
<code>(main)
    while (n<O)
    {
      ...//code inchanger
      if (obj[n].p1.y<0)
      {
        	obj[n] = retour_au_point_de_depart_objet(obj[n], obj[O-n]); // O-n ne prendra jamais 0 (O=0 et nmax=9)
      }
      
    }
</code>
  compilation OK
  execution BUG
  
ECHEC

### test 3

On garde l'appel ds le main et on refait entierement la fonction
Il me semble que le probleme vient de la fonction
  Suppression total du contenu de la fonction

<code>(base de la fonction)
    OBJET retour_au_point_de_depart_objet(OBJET obj, OBJET opposee)
    {
	    if (opposee.p1.x < 720)
	    {
	    	obj.p1.y = 720;
	    	obj.p2.y = obj.p1.y +25;
	    }
	    
	    return obj;
    }
</code>
  compilation OK
  execution BUG liée à la fonction 
  
ECHEC partielle
On va garder ce teste dans le programme et le modifier pour corrigé le bug
